model/common/tests/common/interpolation/unit_tests/test_rbf_interpolation.py

```diff
@@ -209,16 +222,21 @@ def test_rbf_interpolation_coeffs_cell(
 @pytest.mark.level("unit")
 @pytest.mark.datatest
 @pytest.mark.parametrize(
-    "grid_file, experiment, atol",
+    "experiment, atol",
     [
-        (dt_utils.R02B04_GLOBAL, dt_utils.GLOBAL_EXPERIMENT, 3e-10),
-        (dt_utils.REGIONAL_EXPERIMENT, dt_utils.REGIONAL_EXPERIMENT, 3e-3),
+        (definitions.Experiments.EXCLAIM_APE, 3e-10),
+        (definitions.Experiments.MCH_CH_R04B09, 3e-3),
```

Comment:
Is the reason for the bigger error the higher resolution? Wondering if it makes sense to maintain a different threshold? Or maybe we could have a threshold per RnBk?

Reply:
It happens in quite some places that the regional MCH has a higher error, also in the dycore comparision. I think there are several possible reasons for this: 
- resolution (but the R04B09 is higher resolution than the R02B04) 
- boundary effects
- possibly mixed precision, (I do not even know whether the `mch_ch_r04b09` is mixed precision)


I don't know therefore whether a threshold per RnBk would fix it. Generally, there could be the same RnBk for a regional and global grid. 

Generally, if there is no special tol like this in the test we take the looser bound, unless it is really bad like this one.

Reply:
> The mapping could be just:
> {(4,9): x, (2,4): y}, doesn't have to be the full function tol(r,b)

Fair enough. Keep in mind that there will be tests with torus grids soon which would require slightly different handling (no r/b parameters).